packages/edit-post/src/store/actions.js

```diff
@@ -185,17 +185,24 @@ export function* switchEditorMode( mode ) {
 }
 
 /**
- * Returns an action object used to toggle a plugin name flag.
+ * Triggers an action object used to toggle a plugin name flag.
  *
  * @param {string} pluginName Plugin name.
- *
- * @return {Object} Action object.
  */
-export function togglePinnedPluginItem( pluginName ) {
-	return {
-		type: 'TOGGLE_PINNED_PLUGIN_ITEM',
-		pluginName,
-	};
+export function* togglePinnedPluginItem( pluginName ) {
+	const isPinned = yield controls.select(
+		interfaceStore.name,
```

Comment:
Unrelated comment: did we miss controls when introducing the way to use store objects as a reference to `select` and `dispatch` calls?

Reply:
I haven't seen any that use a string literal, so I feel like this was covered. Just checked and the linting also works 👍 

I did wonder whether interface's scope could also be using the store name.

Reply:
I opened #34170. I totally missed data controls when introducing store definitions in #27548.